src/pgwire/src/protocol.rs

```diff
@@ -1140,6 +1149,7 @@ where
         .boxed()
```

Comment:
You could add a `.instrument(...)` call before `.boxed()` here, instead of decorating with `#[instrument ...]`